src/omp/exec_dist.f90

```diff
@@ -127,15 +128,15 @@ subroutine exec_dist_transeq_compact( &
       ! Handle dud by locally generating u*v
       do j = 1, n
         !$omp simd
-        do i = 1, SZ
+        do i = 1, sz
```

Comment:
For example here we basically tell compiler to do its best to vectorise the loop here. But if the loop here goes from 1 to 7 lets say, its impossible for compiler to vectorise this. And if `sz` here is a variable not a parameter, compiler can't be sure at compile time that this is a good integer for vectorisation. In the best case scenario it'll generate multiple versions of assembly, and will have to decide which one to execute only at runtime, which is not ideal. It may alltogether refuse to generate vectorised assembly, and this will depend on the compiler. So here I recommend using `SZ` as a parameter directly from `m_omp_common`.